modules/concom/report/sitesupport/report.js

```diff
@@ -0,0 +1,10 @@
+/* jshint esversion: 6 */
+/* globals Vue */
+
+import reportGeneration from '../../../../sitesupport/reportGeneration.vue.js'
```

Comment:
It feels like these could be DRY'd up by moving them up to general sitesupport, but I also know we're trying to keep things modular, so maybe not?

Reply:
I renamed these files to vue.js because they will be likely used for other vue components as we move forward anyway.
I am for keeping things modular for now, we can DRY them up later if fitting.